### PR TITLE
add  r-ape  r-copula  r-cvtools  r-devtools  r-domc  r-doparallel  r-dplyr  r-e1071

### PR DESCRIPTION
**Pull request by bgruening**
r-ape
 r-copula
 r-cvtools
 r-devtools
 r-domc
 r-doparallel
 r-dplyr
 r-e1071

xref: https://github.com/conda-forge/staged-recipes/issues/2009
ping @johanneskoester

### CODE REVIEW

**Comment by bgruening** [comment]
Green! ping @johanneskoester

File: recipes/r-copula/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    - r-colorspace
+    - r-gsl
+    - r-lattice
+    - r-mvtnorm
+    - r-numderiv
+    - r-pcapp
+    - r-pspline
+    - r-stabledist >=0.6_4
+    - openblas 0.2.19|0.2.19.*
+    - libgcc
```
Comment (johanneskoester) [inline recipes/r-copula/meta.yaml]: You should add `# [not win]` to all libgcc dependencies.

Comment (bgruening) [inline recipes/r-copula/meta.yaml]: Done!

**Comment by bgruening** [comment]
Ready @johanneskoester !

**Pull request merged**